lms/djangoapps/teams/tests/test_views.py

```diff
@@ -122,7 +122,7 @@ def setUpClass(cls):
                     'id': 'topic_{}'.format(i),
                     'name': name,
                     'description': 'Description for topic {}.'.format(i)
-                } for i, name in enumerate([u'sólar power', 'Wind Power', 'Nuclear Power', 'Coal Power'])
+                } for i, name in enumerate([u'Sólar power', 'Wind Power', 'Nuclear Power', 'Coal Power'])
```

Comment:
Is this because the test sqlite3 DB doesn't automatically do case-insensitive ordering for us so we are testing for it?